### ISSUE TITLE
Error in normal mode

### ISSUE THREAD

**Issue by kipelovets** [chenasraf/text-transform.nvim#29]
## Description

I get an error trying to convert word case in normal mode. Seems affect all target cases (camel, snake, etc). Reproduces both with `<leader>~` and commands like `:TtSnake` 

Error message:
```
Error executing vim.schedule lua callback: ...azy/text-transform.nvim/lua/text-transform/replacers.lua:180: attempt to index field 'positions' (a nil value)
stack traceback:
	...azy/text-transform.nvim/lua/text-transform/replacers.lua:180: in function 'get_visual_selection_details'
	...azy/text-transform.nvim/lua/text-transform/replacers.lua:114: in function 'replace_selection'
	.../text-transform.nvim/lua/text-transform/popup/common.lua:61: in function <.../text-transform.nvim/lua/text-transform/popup/common.lua:60>

```

## Steps to reproduce

1. I did the command `:TtCamel` in normal mode while cursor was on a a word "test_test_test"
2. Then I saw error message (see above)
3. Error

## Expected behavior

Word is converted to the desired case

## Environment

- Neovim version: NVIM v0.10.0 from Brew
- text-transform version: latest
- Plugin clash: not sure

**Comment by chenasraf** [comment]
Thanks for the bug report! I'll look into it and see what's going on

**Comment by chenasraf** [comment]
@kipelovets I think I found the problem - can you try using `branch = "develop"` in your setup and tell me if that fixes it for you? Don't forget to update the plugin via your plugin manager using `:Lazy sync` or `:PackerSync`

If it does, I'll put it on master asap.

**Comment by chenasraf** [comment]
Should be fixed with [Release v0.9.1](https://github.com/chenasraf/text-transform.nvim/releases/tag/v0.9.1), if you still have problems, please re-open this issue or create a new one

**Comment by kipelovets** [comment]
Works as a charm, thanks for a quick fix!
